tests/determinism.rs

```diff
@@ -0,0 +1,57 @@
+use anyhow::Result;
+use fs_err::File;
+use rand::rngs::StdRng;
+use rand::SeedableRng;
+use sha2::{Digest, Sha256};
+use spc::{protobuf, Input, Population, MSOA};
+use std::collections::BTreeSet;
+use std::io::{BufRead, BufReader};
+
+// Gets the hex encoded SHA256 hash from a generated population protobuf output given input and rng.
+async fn population_protobuf_hash(input: &Input, dir: &str, mut rng: StdRng) -> Result<String> {
+    let (population, _) = Population::create(input.clone(), &mut rng).await?;
+    std::fs::create_dir_all(dir).unwrap();
+    let output = format!("{dir}/population.pb");
+    protobuf::convert_to_pb(&population, output.clone())?;
+    Ok(hex::encode(Sha256::digest(std::fs::read(output).unwrap())))
+}
+
+#[tokio::test]
```

Comment:
```suggestion
// To run: cargo test --release -- --ignored
#[tokio::test]
```

Saves the next person a moment of googling :)